modules/ROOT/pages/execution-plans/operators.adoc

```diff
@@ -1155,17 +1151,18 @@ Total database accesses: 2, total allocated memory: 184
 
 ======
 
+
+[[query-plan-node-unique-index-seek]]
+== Node Unique Index Seek
+// NodeUniqueIndexSeek
+
 ////
 [source, cypher, role=test-setup]
 ----
-CREATE CONSTRAINT constraint_Team_name IF NOT EXISTS FOR (t:Team) REQUIRE (t.name) IS UNIQUE
+CREATE CONSTRAINT Team_name IF NOT EXISTS FOR (t:Team) REQUIRE (t.name) IS UNIQUE
```

Comment:
Should 'Team' be capitalised? Same question for below example.

Reply:
No strong opinion on this, but makes sense to have it lowercase.